### PR TITLE
Add Studio dev env improvements and docs

### PR DESCRIPTION
**Pull request by elliott-davis**
tenor-25213883
Signed-off-by: Travis Elliott Davis <edavis@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by eeyun** [comment]
Can I ask, could this go into the wiki instead of in the root of the repository, or is this something that needs to live inside the source code?

**Comment by elliott-davis** [comment]
@eeyun I moved it into to BUILDER_DEV. I'm also happy to add it to the wiki if you think it'd be useful there

File: BUILDER_DEV.md [hunk 1]
```diff
@@ -213,6 +213,17 @@ HTTP/1.1 201 Created
...
 
+## Habitat Studio Based Development Environment
+This will start everything to run builder in a studio except the web app
+
+1. `hab studio enter` (This will download and install and start the builder service)
+1. `export HAB_AUTH_TOKEN=<my_token>` (Unless you have one set in your cli.toml)
+1. `sl` Wait until you see the origin service start
+1. `origin`
+1. `keys`
+1. `load_packages`
```
Comment (cnunciato) [inline BUILDER_DEV.md]: If this relies on an actual set of packages somewhere, should we explain where they should go, and link to the archive somehow?

File: BUILDER_DEV.md [hunk 2]
```diff
@@ -213,6 +213,17 @@ HTTP/1.1 201 Created
...
+### Web development
+This is a great environment to use for development on builder-web. Just make sure you're running builder-web locally. Everything should follow the standard web setup.
```
Comment (cnunciato) [inline BUILDER_DEV.md]: We should probably link to the builder-web readme here.

**Comment by cnunciato** [comment]
Nice! Only other suggestions I'd make, then, would be to mention that `direnv` is required (installed and sourced into your shell), a mention of how cli.toml is used, and maybe a tweak to honor environment variables if they're set. Otherwise:

tenor-167359958

**Comment by eeyun** [comment]
I totally wrote a response to this and never posted it because I am a 100% interrupt driven worker :)

I'm happy to have it where you've got it now. My preference would be that we expose more of this type of documentation into the [wiki](https://github.com/habitat-sh/habitat/wiki) moving forward. Just so we can centralize and clean out the `RANDOM.md` files spread throughout the repository. There's (currently) no obligation to do this but I think so long as our README.md makes mention of the wiki and links to wiki articles we'll be providing a better contributor experience by going that path. That, and it's still super easy to clone and change the wiki via CLI if that's your bag.

**Pull request merged**